lib/frameworks/jasmine2.js

```diff
@@ -74,11 +74,29 @@ exports.run = function(runner, specs) {
   var reporter = new RunnerReporter(runner);
   jasmine.getEnv().addReporter(reporter);
 
-  // Filter specs to run based on jasmineNodeOpts.grep and jasmineNodeOpts.invert.
-  jasmine.getEnv().specFilter = function(spec) {
+  // Filter specs/suite to run based on jasmineNodeOpts.grep and jasmineNodeOpts.invert.
+  jasmine.getEnv().specFilter = function(spec, suite) {
+    var suiteMatched = isSuiteMatched(suite);
```

Comment:
we can additional option like jasmineNodeOpts.ignoreSuite, jasmineNodeOpts.ignoreSpec. What is your thought to support this?